03/app.js

```diff
@@ -0,0 +1,62 @@
+const fileEl = document.querySelector('.files input');
+const filesList = [];
+
+fileEl.addEventListener('change', getFile);
+
+function getFile(e) {
+    const inputFilesList = e.target.files;
+
+    for (let i = 0; i < inputFilesList.length; i++) {
+        const file = inputFilesList[i];
+
+        if (file && file.type.includes('image')) {
+            const reader = new FileReader();
+
+            reader.onload = function (readerEvent) {
+                const newImg = document.createElement('img');
+                newImg.src = readerEvent.target.result;
+
+                const newFile = new ImgFile(file, newImg);
+
+                filesList.push(newFile);
+            }
+            reader.readAsDataURL(file);
+        } else {
+            alert('Plik musi być obrazkiem!');
+        }
+    };
+}
+
+const ImgFile = function (initFile, initNewImg) {
+    this.fileName = initFile.name;
+    // Nie jestem pewien co do wartości, bo teoretycznie o ile dobrze pamiętam powinno się to dzielić przez 1024, ale wtedy wychodzi inna wartość niż rozmiar obrazka, który podpowiada mi system
```

Comment:
Zdecydowanie powinno być przez 1024 - możliwe, że system podpowiada ile on zajmuje na dysku (a nie jaki ma faktycznie rozmiar) - może o to chodzi?